package.json

```diff
@@ -2,7 +2,7 @@
     "name": "@vimeo/iris",
     "homepage": "https://github.com/vimeo/iris/tree/main",
     "bugs": "https://github.com/vimeo/iris/issues",
-    "version": "0.115.0",
+    "version": "0.116.0",
```

Comment:
Is the version bump automated?

Reply:
No, it is not. We do it manually at the time of release. It can't be done in PRs because the order of PRs being merged and how that coincides with release timing can't be known in advance. Please remove this change.